__app__/adlistingparser/sites/sumosear_ch.py

```diff
@@ -0,0 +1,44 @@
+from urllib.parse import urljoin, urlparse
+import re
+from bs4 import BeautifulSoup
+from __app__.adlistingparser.sites.base_adlisting_parser import (
+    BaseAdListingParser,
+    AdListing,
+)
+
+
+class SumoSear_ch(BaseAdListingParser):
+    def ad_listings(self):
+        ad_listings = []
+        date = "Not found"
+        for item in self.soup.findAll("a", class_="img-res-item"):
+                url = item.get("href")
+                metadata = {}
+                time = item.find("time", class_="img-res-item__time").get_text()
+                #Advertisers sometimes put their phone numbers or weird comments in the location section, so the results for this section of the parser aren't perfect.
+                location = item.find("div", class_="img-res-item__attr_txt").get_text()
+                if time:
+                    metadata["ad-date-posted"] = time
+                if location:
+                    metadata["location"] = location
```

Comment:
```suggestion
                if time:
                    metadata["ad-date-posted"] = time
```

Same as my comment below. We've been having trouble with weird text in location strings.